doc/sphinx/proofs/writing-proofs/inductive.rst

```diff
@@ -186,56 +195,52 @@ analysis on inductive or co-inductive objects (see :ref:`inductive-definitions`)
       clause is an occurrence clause whose syntax and behavior is described
       in :ref:`occurrences sets <occurrencessets>`.
 
-   .. tacv:: destruct @term {? with @bindings } {? as @or_and_intropattern_loc } {? eqn:@naming_intropattern } {? using @term {? with @bindings } } {? in @goal_occurrences }
-             edestruct @term {? with @bindings } {? as @or_and_intropattern_loc } {? eqn:@naming_intropattern } {? using @term {? with @bindings } } {? in @goal_occurrences }
+   .. tacv:: destruct @term {? with @bindings } {? @as_or_and_ipat } {? eqn:@naming_intropattern } {? using @term {? with @bindings } } {? in @goal_occurrences }
+             edestruct @term {? with @bindings } {? @as_or_and_ipat } {? eqn:@naming_intropattern } {? using @term {? with @bindings } } {? in @goal_occurrences }
 
       These are the general forms of :tacn:`destruct` and :tacn:`edestruct`.
       They combine the effects of the ``with``, ``as``, ``eqn:``, ``using``,
       and ``in`` clauses.
 
-.. tacn:: case @term
-   :name: case
+.. tacn:: case @induction_clause_list
 
    The tactic :n:`case` is a more basic tactic to perform case analysis without
-   recursion. It behaves as :n:`elim @term` but using a case-analysis
-   elimination principle and not a recursive one.
+   recursion. It behaves like :n:`elim @term` but uses a case-analysis
+   elimination principle rather than a recursive one.
```

Comment:
Re:
```
.. tacn:: case @induction_clause_list   
The tactic :n:`case` is a more basic tactic to perform case analysis without
recursion. It behaves as :n:`elim @term` but uses a case-analysis
elimination principle and not a recursive one.
```

I'd like to understand this better because `simple destruct` is defined in terms of `case`.  Perhaps you can point me to a clear example?

Reply:
The table to have in mind is:

|    |      Old (no `intros`, no `as`)      |  New (does `intros`, `as` clause) |
|----------|:-------------:|:-----:|
| no induction hypothesis (inserts a `match`) |  case | destruct |
| induction hypothesis (uses induction principle) |    elim   |   induction |

Beyond this, there are subtleties of implementation that I'm not knowledgeable about.

Reply:
> Old (no `intros`, no `as`)

Also does not change uses of the variable in the context.

Reply:
The table is great as a mental model, though I don't feel it's necessary to include it as a table.  The syntax differences will be apparent and in the text I do label the old tactics as such, explain which use induction hypotheses, and which modify the local context.